arcgis-ios-sdk-samples/Layers/Web tiled layer/WebTiledLayerViewController.swift

```diff
@@ -0,0 +1,68 @@
+//
+// Copyright 2016 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import UIKit
+import ArcGIS
+
+class WebTiledLayerViewController: UIViewController {
+
+    @IBOutlet private var mapView:AGSMapView!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        //add the source code button item to the right of navigation bar
+        (self.navigationItem.rightBarButtonItem as! SourceCodeBarButtonItem).filenames = ["WebTiledLayerViewController"]
+
+        //url template for web tiled layer
+        let urlTemplate = "http://{subDomain}.tile.stamen.com/terrain/{level}/{col}/{row}.png"
+        
+        //sub domains
+        let subDomains = ["a", "b", "c", "d"]
+        
+        //attribution
+        let attribution = "Map tiles by <a href=\"http://stamen.com/\">Stamen Design</a>, "
```

Comment:
Can we leave out the links till we properly support them in the attribution bar?